src/components/common/GreenhouseEffectsSettings.js

```diff
@@ -0,0 +1,49 @@
+import { Typography } from '@material-ui/core';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { useDispatch, useSelector } from 'react-redux';
+import SliderWithLabel from './SliderWithLabel';
+import { setGreenhouseGasesValues } from '../../actions';
+
+const GreenhouseEffectsSettings = () => {
+  const { t } = useTranslation();
+  const dispatch = useDispatch();
+  const values = useSelector(({ lab }) => lab.greenhouseGasesValues);
+  const { ch4, co2, h2o } = values;
```

Comment:
can we use _methane_, _carbonDioxide_, _water_? feel like it's easier/safer than having numbers in the variable names...